src/map.js

```diff
@@ -0,0 +1,233 @@
+import React from "react";
+import L from "leaflet";
+import borderData from "./border.js";
+import leafletPip from "@mapbox/leaflet-pip";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      markerPosition: { lat: 44.0286, lng: -72.7317 },
+      oldMarkerPosition: { oldLat: 44.0886, oldLng: -72.7317 },
+      minimumZoom: 18,
+      maximumZoom: 18,
+      defaultZoom: 18,
+      gameStarted: false,
+      borderData: L.geoJSON(borderData),
+      currentPoint: { lat: 44.0286, lng: -72.7317 },
+      randomCord: { lat: 44.0286, lng: -72.7317 }
+    };
+
+    // binding the functions to the state
+    this.moveMapNorth = this.moveMapNorth.bind(this);
+    this.moveMapSouth = this.moveMapSouth.bind(this);
+    this.moveMapEast = this.moveMapEast.bind(this);
+    this.moveMapWest = this.moveMapWest.bind(this);
+
+    this.startGame = this.startGame.bind(this);
+    this.quitGame = this.quitGame.bind(this);
+
+    this.randomCord = this.randomCord.bind(this);
+
+    this.randomLat = this.randomLat.bind(this);
+    this.randomLong = this.randomLong.bind(this);
+  }
+
+  // startGame function generates a random point within the declared boundaires 
+  startGame() {
+    this.setState({
+      gameStarted: true
+    });
+    if (this.state.gameStarted) {
```

Comment:
Since you're already explicitly setting `gameStarted` to 'true` you don't actually need to have an if statement here. You could just move directly to calling `this.randomCord()` after setting the state.